src/helpers.js

```diff
@@ -65,6 +66,34 @@ export default {
       allureReporter.addAttachment('Execution video', videoPath, 'video/mp4');
     }
 
+    const frames = glob.sync(`${this.recordingPath}/*.png`);
+
+    if (frames.length) {
+      const frameRegex = /^.*\/(\d{4})\.png$/;
+      const frameNumbers = frames.map((path) => +path.replace(frameRegex, '$1'));
+      const pad = (frameNumber) => frameNumber.toString().padStart(4, '0');
+      const insertMissing = (sourceFrame, targetFrame) => {
+        const src = `${this.recordingPath}/${pad(sourceFrame)}.png`;
+        const dest = `${this.recordingPath}/${pad(targetFrame)}.png`;
+        const options = {overwrite: false};
+        fs.copySync(src, dest, options);
+      };
+
+      if (frameNumbers.length !== frameNumbers[frameNumbers.length - 1] - frameNumbers[0] + 1) {
+        // fill in any blanks
+        let nextFrame;
+        let lastFrame;
+        for (let i = frameNumbers[0]; i < frameNumbers[frameNumbers.length - 1]; ++i) {
+          if (nextFrame && !frameNumbers.includes(i)) {
+            insertMissing(lastFrame, i);
```

Comment:
Let's add a log statement informing the user this happened

Reply:
I actually took that out before committing!